### ISSUE TITLE
SingleDatePicker doesn't work within a form with other fields

### ISSUE THREAD

**Issue by dkwin** [airbnb/react-dates#105]
Hey guys

```
I'm using SingleDatePicker along side with some bootstrap forms and it seems that the SingleDatePicker doesn't render fully.
```

I'm using React .15 with Moment 2.14 and React Dates 3.1.1

OnFocus

![react-datebug2](https://cloud.githubusercontent.com/assets/3934306/19060040/b34748c0-89b2-11e6-80d9-fbe6c85de79e.gif)

After Clicking Calendar

![react-datebug](https://cloud.githubusercontent.com/assets/3934306/19060014/79e279e2-89b2-11e6-8359-2bf355f1d0cc.gif)

**Comment by majapw** [comment]
Hmm, I notice that you are using tabs on this interface. Are you controlling your tabs by rendering each time or are you showing/hiding them with `display: none` when they are hidden?

If you are hiding them with CSS, I wonder if this is related to https://github.com/airbnb/react-dates/issues/46.

**Comment by bsr203** [comment]
has the same issue using with bootstrap 4. Once clicked the Calendar everything seems ok.
I render without a tab.

**Comment by tvanro** [comment]
We also encountered the same issue and upgrading react-dates from 3.3.1 to 3.3.2 resolved it.
I'm pretty sure it's because of the tether library that is added in #83

**Comment by savovs** [comment]
Having a similar issue when using DayPicker in a form with tabs that are hidden with `display: none`.
[http://i.imgur.com/qM9uxAv.gifv](url)

**Comment by majapw** [comment]
I believe this should be fixed in v4.1.1. 

@dkwin @bsr203 @tvanro @savovs Please check it out and let me know if this is not the case.
